marlowe-playground-client/src/Marlowe/Holes.purs

```diff
@@ -1,3 +1,4 @@
+-- TODO: Rename to Marlowe.Term
```

Comment:
@palas, what do you think about this?

Reply:
I kind of prefer `Holes`, even though is not accurate, `Term` is quite a generic "term" (see what I did there?)